src/components/dls/QuranWord/QuranWord.tsx

```diff
@@ -19,23 +21,30 @@ const getGlyph = (word: Word, font: QuranFont) => {
 };
 
 const QuranWord = ({ word, font, highlight }: QuranWordProps) => {
-  let wordText;
+  const { showWordByWordTranslation, showWordByWordTransliteration } =
+    useSelector(selectReadingPreferences);
+  const isWordByWordLayout = showWordByWordTranslation || showWordByWordTransliteration;
+  let wordText = null;
 
   if (isQCFFont(font)) {
     wordText = <GlyphWord font={font} text={getGlyph(word, font)} pageNumber={word.pageNumber} />;
-  } else {
-    wordText =
-      word.charTypeName === CharType.Pause ? (
-        ''
-      ) : (
-        <TextWord font={font} text={word.text} charType={word.charTypeName} />
-      );
+  } else if (word.charTypeName !== CharType.Pause) {
+    wordText = <TextWord font={font} text={word.text} charType={word.charTypeName} />;
   }
 
   return (
-    <span className={classNames(styles.container, { [styles.highlighted]: highlight })}>
+    <div
+      className={classNames(styles.container, {
+        [styles.highlighted]: highlight,
+        [styles.wbwContainer]: isWordByWordLayout,
+      })}
+    >
       {wordText}
-    </span>
+      {showWordByWordTransliteration && (
+        <p className={styles.wbwText}>{word.transliteration?.text}</p>
+      )}
+      {showWordByWordTranslation && <p className={styles.wbwText}>{word.translation?.text}</p>}
+    </div>
```

Comment:
One issue that I see here is that we'll likely have different styles for wbw translations and wbw transliteration. that's easy to fix in the polish stage though. No need to address in this PR